scripts/holidays2json.js

```diff
@@ -62,9 +63,18 @@ Holidays2json.prototype = {
     const obj = this.load('0')
     obj.holidays = {}
     this.list.forEach(function (cc) {
-      Object.assign(obj.holidays, this.load(cc).holidays)
+      const data = this.load(cc)
+      fs.writeFile(resolve(config.jsonDir, `${cc}.json`), JSON.stringify(data), 'utf8', err => {
```

Comment:
These writeFile calls feel out of place in the build function given that there is a save function already defined.